railties/lib/rails/application/configuration.rb

```diff
@@ -210,6 +210,10 @@ def load_defaults(target_version)
             active_support.hash_digest_class = OpenSSL::Digest::SHA256
             active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
           end
+
+          if respond_to?(:action_mailer)
+            action_mailer.default_smtp_settings = { open_timeout: 5, read_timeout: 5 }
```

Comment:
I think we should call this `default_smtp_timeout_value` and set it to be 5. If people want a different value they can use `action_mailer.smtp_settings`